### ISSUE TITLE
SpiecEasi on conda

### ISSUE THREAD

**Issue by sghignone** [zdk123/SpiecEasi#234]
Hi!
Has anyone managed to install SpiecEasi in a conda environment?
Can you tell specification of your environment? 
thanks
S.-

**Comment by zdk123** [comment]
Yes, there is a recipe on bioconda: https://anaconda.org/bioconda/r-spieceasi

**Comment by sghignone** [comment]
yes, I know the recipe exists, but I'm getting this..(I thought conda was able to solve those dependencies)

$ conda install -c bioconda r-spieceasi
Retrieving notices: ...working... done
Collecting package metadata (current_repodata.json): done
Solving environment: failed with initial frozen solve. Retrying with flexible solve.
Solving environment: failed with repodata from current_repodata.json, will retry with next repodata source.
Collecting package metadata (repodata.json): done
Solving environment: failed with initial frozen solve. Retrying with flexible solve.
Solving environment: | 
Found conflicts! Looking for incompatible packages.
This can take several minutes.  Press CTRL-C to abort.
failed                                                                                                                             

UnsatisfiableError: The following specifications were found to be incompatible with your system:

  - feature:/linux-64::__glibc==2.31=0
  - r-spieceasi -> r-rcpp -> __glibc[version='>=2.17,<3.0.a0']

Your installed version is: 2.31

Note that strict channel priority may have removed packages required for satisfiability.

**Comment by zdk123** [comment]
I'm not sure why your system glibc doesn't satisfy that version
requirement... The version strings look compatible at least. I'll see if I
can reproduce this.
